itou/templates/layout/base.html

```diff
@@ -37,6 +37,9 @@
         <link rel="stylesheet" href="{% static "vendor/jquery-ui/jquery-ui.min.css" %}" type="text/css">
         <link rel="stylesheet" href="{% static "vendor/theme-inclusion/stylesheets/app.css" %}" type="text/css">
         <link rel="stylesheet" href="{% static "css/itou.css" %}" type="text/css">
+        {% if not debug %}
+            <script src="https://js.sentry-cdn.com/d6b350eab2a54e5fab388bade8c9bbad.min.js" crossorigin="anonymous"></script>
```

Comment:
Est-ce qu'on ne voudrait pas le vendor comme le reste de notre JS ?
Et il n'y a pas besoin d'appeler `Sentry.init` avec notre DSN ? https://docs.sentry.io/platforms/javascript/#configure

Reply:
Nope, en fait le loader est un script magique. Sans regarder de trop près, `d6b350eab2a54e5fab388bade8c9bbad` est un hash magique qui est associé chez Sentry à des options pour le script chargé. Ça met à jour la version du SDK tout seul, on peut choisir dans Sentry d’activer le monitoring de perfs, le session replay (j’ai mis non, trop intrusif point de vue données).
https://docs.sentry.io/platforms/javascript/install/loader/

> **SDK Configuration**
   While the Loader Script will work out of the box without any configuration in your application, you can still configure the SDK according to your needs.

Reply:
:+1: 
Après, ce serait le premier script qui utiliserait un CDN.

Reply:
Ça pourrais valoir le coup de mettre un petit commentaire indiquer ça au prochain qui pourraient passer par là, car effectivement ça fait très exception et donc bizarre.

Reply:
Commentaire ajouté :+1: